functions/dialogflow/functions/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "scripts": {
+    "lint": "eslint .",
+    "serve": "firebase serve --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log",
+    "local-test": "mocha test/index.test.js",
+    "pretest": "sh test/updateFunctions.sh",
+    "test": "npm run local-test"
+  },
+  "dependencies": {
+    "firebase-admin": "~7.0.0",
```

Comment:
Can we use `^7.0.0`?

Reply:
I am not exactly sure on this @fhinkel but may be we can.